arbitrum-docs/launch-orbit-chain/bold-adoption-for-orbit-chains.mdx

```diff
@@ -0,0 +1,106 @@
+content_type: release announcement
+author: leeederek
+sme: leeederek
+target_audience: 'Developers who want to build on Arbitrum using popular programming languages, like Rust'
+sidebar_position: 1
+---
+
+:::info PUBLIC PREVIEW DOCUMENT
+
+This document is currently in **public preview** and may change significantly as feedback is captured from readers like you. Click the _Request an update_ button at the top of this document or [join the Arbitrum Discord](https://discord.gg/arbitrum) to share your feedback.
+
+:::
+
+
+## Launch details and key dates
+
+- **Status:** Alpha - continued testing and evaluation of performance of BoLD with Stylus
+- **Sepolia** Mid August 2024                                             
+- <a data-quicklook-from="arbitrum-one">**Arbitrum One**</a> Late Sept 2024
+- <a data-quicklook-from="arbitrum-nova">**Arbitrum Nova**</a> Late Sept 2024
+
+
+### tldr;
+
+Arbitrum BoLD is an upgrade to the existing dispute protocol used by Arbitrum chains that enables permissionless validation and other security benefits. Similar to other new features for the Arbitrum technology stack, BoLD will be made available for mainnet adoption by Orbit chains when they are ready. Specifically for Arbitrum BoLD, an upgrade will involve a Nitro node software version bump and the deployment and consumption of new smart contracts on the parent chain.
+
+Adopting <a data-quicklook-from="bold">Arbitrum BoLD</a> brings new security benefits to Orbit chains that are independent of whether or not validation is permissionless. These new security benefits include improved resistance to delay attacks and increased censorship resistance for `L3s`. Because of these new security benefits, we strongly recommend teams to upgrade their Orbit chains to use Arbitrum BoLD (for the security benefits) but **keep validation permissioned**. Below is a quick breakdown of the impacts of permissioned BoLD vs. permissionless BoLD on your Orbit chain:
+
+
+#### Permissioned BoLD on an Orbit chain
+
+##### Benefits                                                                                                           
+    - Improved resistance to delay attacks                                                                               
+    - Increased censorship resistance for L3s                                                                            
+    - Use of any `ERC-20` as the bonding token                                                                           
+    - Be on the latest iteration of Arbitrum technology, making future security & feature updates much easier to pick up 
+    - Less engineering overhead and infrastructure costs                                                                 
+
+##### Caveats 
+    - Permissioned validation: only entities on allowlist can resolve disputes
```

Comment:
might add as a benefit that you get 1/N honest assumption

Reply:
Makes sense - though this is true even without BoLD. I will add it though.